### ISSUE TITLE
grad_outputs for gradient penalty

### ISSUE THREAD

**Issue by leesunfreshing** [caogang/wgan-gp#28]
Hi, could you specify the purpose of grad_outputs? It is not clear given the PyTorch Tutorial... Moreover, why it should be torch.ones for penalty item?

**Comment by caogang** [comment]
The grad_outputs is backward gradients passed from the upper-layer when performing backward procedure. This item is torch.ones because that it is the last layer, no more layer after.
